modyn/tests/evaluator/internal/metrics/test_roc_auc.py

```diff
@@ -39,6 +39,12 @@ def test_roc_auc_metric():
     assert get_evaluation_result(y_true, y_score) == pytest.approx(2 / 3)
 
 
+def test_roc_auc_with_two_entries():
+    y_true = np.array([0, 1])
+    y_score = np.array([0.1, 0.6])
+    get_evaluation_result(y_true, y_score)
```

Comment:
without the change, this line will raise an exception

Reply:
thank you, now I understand the issue!